anta/device.py

```diff
@@ -446,6 +438,18 @@ async def _collect(self, command: AntaCommand, *, collection_id: str | None = No
             anta_log_exception(e, f"An error occurred while issuing an eAPI request to {self.name}", logger)
         logger.debug("%s: %s", self.name, command)
 
+    def _log_eapi_command_error(self, command: AntaCommand, e: asynceapi.EapiCommandError) -> None:
+        """Appropriately log the eapi command error."""
+        command.errors = e.errors
+        if command.requires_privileges:
+            logger.error("Command '%s' requires privileged mode on %s. Verify user permissions and if the `enable` option is required.", command.command, self.name)
+        if not command.supported:
+            logger.debug("Command '%s' is not supported on '%s' (%s)", command.command, self.name, self.hw_model)
+        elif command.returned_known_eos_error:
+            logger.debug("Command '%s' returned a known error '%s'", command.command, self.name)
```

Comment:
Not sure if you agree but to me these debug logs are irrelevant:
![image](https://github.com/user-attachments/assets/b4dba418-cfb4-4c72-bd1f-d95d8d3d8148)

debug logs should provide more information and context versus regular logs. That's actually the opposite here, we have less information in debug logs :P

Reply:
I dont think regular logs bring more context here. Maybe you meant to say the error message?